### ISSUE TITLE
android : not redirect to card page

### ISSUE THREAD

**Issue by MustafaSaad45** [Magints-Technology-Solutions/payfort#17]
Hi , how are everybody i work on this plugins in iOS it work after add sdk_token redirect me to card screen 
but in android not redirect me to card page and throw exception technical problem

**Comment by MAdnan-ST** [comment]
@MustafaSaad45 did you solve this issue?

**Comment by MustafaSaad45** [comment]
@MAdnan-ST yes bro, but it does not work on all devices. but the error in device Id generation

**Comment by mariamalbarghouti** [comment]
the same issue

**Comment by MustafaSaad45** [comment]
@mariamalbarghouti can you take screenShot of your code to look if i can help you?
